pyproject.toml

```diff
@@ -139,3 +140,19 @@ convention = "pep257"
     "D100",
     "D103",
 ]
+
+[tool.mypy]
+warn_unused_configs = true  # warns if overrides sections unused/mis-spelled
+
+[[tool.mypy.overrides]]
+module = [ "kasa.tests.*", "devtools.*" ]
+disable_error_code = "annotation-unchecked"
+
+[[tool.mypy.overrides]]
+module = [
+    "devtools.bench.benchmark",
+    "devtools.parse_pcap",
+    "devtools.perftest",
+    "devtools.create_module_fixtures"
+]
+disable_error_code = "import-not-found,import-untyped"
```

Comment:
All these modules have dependencies not configured in `pyproject.toml`. We may want to consider adding these as an extra `devtools`?

Reply:
I see no harm doing that, but it isn't really that useful as these are only useful for developers & it would cause `--all-extras` to install things like pandas which is not that small for a library.